README.md

```diff
@@ -101,7 +101,34 @@ would in a Django shell.
 
 Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
 
-    docker-compose run --rm web pytest
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
+    
+### If you do not already have a running container, you should replace ```exec``` with ```run --rm```
+    docker-compose exec web pytest
+    # Run Python tests in a single file
+    docker-compose exec web pytest /path/to/test.py
+    # Run Python test cases in a single file that match some function/class name
+    docker-compose exec web pytest /path/to/test.py -- -k test_some_logic
```

Comment:
Please change to `docker-compose exec web pytest /path/to/test.py -k test_some_logic`